sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs

```diff
@@ -278,16 +285,20 @@ private static void AddTelemetryFromActivityEvents(Activity activity, TelemetryI
                     exceptionType = tag.Value?.ToString();
                     continue;
                 }
-                if (tag.Key == SemanticConventions.AttributeExceptionMessage)
+                else if (tag.Key == SemanticConventions.AttributeExceptionMessage)
                 {
                     exceptionMessage = tag.Value?.ToString();
                     continue;
                 }
-                if (tag.Key == SemanticConventions.AttributeExceptionStacktrace)
+                else if (tag.Key == SemanticConventions.AttributeExceptionStacktrace)
                 {
                     exceptionStackTrace = tag.Value?.ToString();
                     continue;
                 }
+                else
+                {
+                    TryAddKvp(properties, tag);
```

Comment:
https://github.com/Azure/azure-sdk-for-net/blob/5c83c56d55343d3db9e245611d7715712531e6ed/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs#L262  --> this does not seem to use this helper, and hence missing the key-length/null-value validations?

Reply:
Good catch! I'll change this to use the new helper. Thanks!